dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, options=None):
         self.sum = 0
         self.variance = 0
         self.max_histogram_bin = 10000
```

Comment:
Is it worth increasing this?

Reply:
I need to re-evaluate time, but maybe? I think the next refactor (which i would offload) would make it feasible to be higher.